crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1088,6 +1100,24 @@ impl CumulocityConverter {
         Ok(converted_messages)
     }
 
+    fn append_id_if_not_given(
+        &mut self,
+        register_message: &EntityRegistrationMessage,
+        source: &EntityTopicId,
+    ) -> Vec<MqttMessage> {
```

Comment:
On all the sites this method is called with a source that is `register_message.topic_id`.

```suggestion
    fn append_id_if_not_given(
        &mut self,
        register_message: &EntityRegistrationMessage,
    ) -> Vec<MqttMessage> {
    let source = &register_message.topic_id;
```

Reply:
done in [02caf34](https://github.com/thin-edge/thin-edge.io/pull/2925/commits/02caf346e9135d3deb2fe09feff5a1a23987a807)